internal/encoding/merge/parquet.go

```diff
@@ -16,7 +19,20 @@ import (
 )
 
 // ToParquet merges multiple blocks together and outputs a key and merged Parquet data
-func ToParquet(blocks []block.Block, schema typeof.Schema) ([]byte, error) {
+func ToParquet(input interface{}) ([]byte, error) {
```

Comment:
I'm a bit wary of removing schema. Semantically taking a schema from the first block is different from providing a schema to the function itself. Why do we need to remove schema?